en/08_Changelogs/5.4.0.md

```diff
@@ -411,6 +411,17 @@ The exception was made for the following reasons:
 - [`CMSSiteTreeFilter::mapIDs()`](api:SilverStripe\CMS\Controllers\CMSSiteTreeFilter::mapIDs()) has been deprecated. It will be removed without equivalent functionality to replace it.
 - [`ElementSiteTreeFilterSearch`](api:DNADesign\Elemental\Controllers\ElementSiteTreeFilterSearch) has been deprecated. It will be replaced with `DNADesign\Elemental\ORM\Search\ElementalSiteTreeSearchContext`.
 - [`ElementalCMSMainExtension`](api:DNADesign\Elemental\Extensions\ElementalCMSMainExtension) has been deprecated. It will be removed without equivalent functionality to replace it.
+- The [`HTMLEditorField.fixed_row_height`](api:SilverStripe\Forms\HTMLEditor\HTMLEditorField->fixed_row_height) configuration property has been deprecated. It will be replaced with `SilverStripe\Forms\HTMLEditor\HTMLEditorConfig.fixed_row_height`.
+- [`HTMLEditorSanitiser::patternToRegex()`](api:SilverStripe\Forms\HTMLEditor\HTMLEditorSanitiser::patternToRegex()) has been deprecated. It will be replaced with `SilverStripe\Forms\HTMLEditor\HTMLEditorRuleSet::patternToRegex()`.
+- [`HTMLEditorSanitiser::addValidElements()`](api:SilverStripe\Forms\HTMLEditor\HTMLEditorSanitiser::addValidElements()) has been deprecated. It will be replaced with `SilverStripe\Forms\HTMLEditor\HTMLEditorRuleSet`.
+- [`HTMLEditorSanitiser::getRuleForElement()`](api:SilverStripe\Forms\HTMLEditor\HTMLEditorSanitiser::getRuleForElement()) has been deprecated. It will be replaced with `SilverStripe\Forms\HTMLEditor\HTMLEditorRuleSet::getRuleForElement()`.
+- [`HTMLEditorSanitiser::getRuleForAttribute()`](api:SilverStripe\Forms\HTMLEditor\HTMLEditorSanitiser::getRuleForAttribute()) has been deprecated. It will be replaced with logic in `SilverStripe\Forms\HTMLEditor\HTMLEditorElementRule`.
+- [`HTMLEditorSanitiser::elementMatchesRule()`](api:SilverStripe\Forms\HTMLEditor\HTMLEditorSanitiser::elementMatchesRule()) has been deprecated. It will be replaced with `SilverStripe\Forms\HTMLEditor\HTMLEditorRuleSet::isElementAllowed()`.
+- [`HTMLEditorSanitiser::attributeMatchesRule()`](api:SilverStripe\Forms\HTMLEditor\HTMLEditorSanitiser::attributeMatchesRule()) has been deprecated. It will be replaced with `SilverStripe\Forms\HTMLEditor\HTMLEditorElementRule::isAttributeAllowed()`.
+- [`TinyMCECombinedGenerator`](api:SilverStripe\Forms\HTMLEditor\TinyMCECombinedGenerator) has been deprecated. It will be replaced with `SilverStripe\TinyMCE\TinyMCECombinedGenerator`.
```

Comment:
I've recommended we change the namespace in the new module to from `\TinyMCE\` to `\HtmlEditorTinyMCE\` [here](https://github.com/silverstripe/silverstripe-htmleditor-tinymce/pull/1#discussion_r1988325388) 

Applies to any other references to the namespace in this PR